### PR TITLE
fix flanking errors and add new test

### PR DESCRIPTION
**Pull request by rnmitchell**
I recently noticed incorrect bracketing annotation for some of the 5' flanking ForenSeq sequences. For example, for vWA, what should have been annotated as `AAT ACAT AGGA [TGGA]2` was annotated as `AAT ACAT AGGA 2]TGGA[`.  This PR is to fix this issue as well as to add a new test to ensure this type of bracketing does not happen again.

### CODE REVIEW

**Comment by rnmitchell** [comment]
Ok, this is ready for review. I'll request @dsommer since @standage is out of town. Thanks!

File: lusSTR/repeat.py [hunk 1]
```diff
@@ -49,18 +49,22 @@ def collapse_all_repeats(sequence, repeats):
     return collapsed_seq
 
 
-def split_by_n(sequence, n):
+def split_by_n(sequence, n, rev):
```
Comment (dsommer) [inline lusSTR/repeat.py:52]: Would it make sense to set the default value for rev=True?

File: lusSTR/repeat.py [hunk 2]
```diff
@@ -49,18 +49,22 @@ def collapse_all_repeats(sequence, repeats):
...
+        if rev is False:
+            yield sequence[:n]
+            sequence = sequence[n:]
+        else:
+            yield sequence[-n:]
+            sequence = sequence[:-n]
 
 
-def get_blocks(sequence, n):
+def get_blocks(sequence, n, rev):
```
Comment (dsommer) [inline lusSTR/repeat.py:63]: Same here, a default value of True for rev might be a good option.

File: lusSTR/repeat.py (hunk 1 above)
Comment (dsommer) [reply lusSTR/repeat.py:52]: Good implementation of split_by_n. I am surprised a function like this doesn't already exist in python but I couldn't find it. This is something I could see being reused in other projects.

Comment (dsommer) [reply lusSTR/repeat.py:52]: I guess the default value should by False not True.

Comment (rnmitchell) [reply lusSTR/repeat.py]: I added default value of False.

File: lusSTR/repeat.py (hunk 2 above)
Comment (rnmitchell) [reply lusSTR/repeat.py]: I added a default value of False.

**Pull request merged**